.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+repos:
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: detect-aws-credentials
+      - id: detect-private-key
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-json
+      - id: check-yaml
+      - id: check-added-large-files
+      - id: check-merge-conflict
+
+  ## Used to find secrets throughout the repository.
+  -   repo: https://github.com/ibm/detect-secrets
+      rev: 0.13.1+ibm.50.dss # https://github.com/Yelp/detect-secrets/releases
+      hooks:
+      -   id: detect-secrets
+          args: ['--baseline', '.secrets.baseline']
+
+  - repo: https://github.com/hadolint/hadolint
+    rev: v2.10.0
+    hooks:
+      - id: hadolint
+
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+    -   id: black
```

Comment:
Is there a way to run a local command line?  We already have a way to do this linting (and more) locally.  `pipenv run nox -s black`, `pipenv run nox -s lint`, and ` yarn lint`.

I rather run the same thing we would run locally.

Reply:
I guess it simply depends on personal usability, this simply forces black to run, if we're not doing any extra black-specific config, then we can go either way.

Reply:
updated pre-commit to use `pipenv run nox -s black`